### ISSUE TITLE
Unexpected error when listObject has no results

### ISSUE THREAD

**Issue by huxwfun** [shimohq/awos-js#1]
code like this (where prefix '1234' does not exist in the oss bucket)
```
client.listObject('123', { prefix: '1234' }).then((r) => {
  console.log(r)
})
```
causes an error
```
(node:89655) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'map' of undefined
    at OSSClient.<anonymous> (/Users/shawnhu/projects/sas/svc-sas/node_modules/awos-js/lib/oss.js:134:32)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/shawnhu/projects/sas/svc-sas/node_modules/awos-js/lib/oss.js:4:58)
    at process._tickCallback (internal/process/next_tick.js:68:7)


return res.objects.map((o) => o.name);
```
It should be something like
```
return res.objects ? res.objects.map((o) => o.name) : [];
```
Haven't tested on AWS

**Comment by huxwfun** [comment]
stupid ali-oss api
return null for a list request

**Comment by NeoyeElf** [comment]
fix the bug in v1.0.1
